src/test/java/seedu/address/logic/commands/DeleteNoteCommandTest.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_STARTUP;
+import static seedu.address.testutil.TypicalStartups.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.startup.Startup;
+import seedu.address.testutil.StartupBuilder;
+
+
+
+public class DeleteNoteCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_deleteNoteUnfilteredList_success() {
+        DeleteNoteCommand deleteNoteCommand = new DeleteNoteCommand(INDEX_FIRST_STARTUP, 1);
```

Comment:
Should we have some negative test cases or edit commands where we add blank screens / numbers etc?

Reply:
Negative test cases have been added. Blank notes are addressed at the NoteTest level.